src/components/appbar.tsx

```diff
@@ -49,6 +52,9 @@ export default function AppBar(props: AppBarProps): React.ReactElement {
         <IconButton id="show-settings-btn" color="inherit" onClick={() => showSettings(true)}>
           <SettingsIcon />
         </IconButton>
+        <IconButton id="show-tour-btn" color="inherit" onClick={() => showTour(true)}>
+          <FontAwesomeIcon icon={faMapSigns} />
+        </IconButton>
```

Comment:
Oh that's what the tutorial there is for? I wasn't sure if it was something else! But sure, that would be better I think, rather than a stand-alone icon on the appbar.